### DISCUSSION TITLE
Welcome to the Community!

### DISCUSSION THREAD

**Discussion by ozgune** [ubicloud/ubicloud#133] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sebastian** [comment]
Hi 👋🏻

super excited about ubicloud and looking forward to seeing how it develops!

I am an ex-founder (real-time anonymizing database engine), currently an engineering manager at a large online retailer, and hope to one day start another company again. 

Ubicloud seems to perfectly address my need for easily slicing up larger bare metal machines hosted at Hetzner! 🎉
